sdcm/results_analyze/__init__.py

```diff
@@ -222,11 +222,12 @@ class LatencyDuringOperationsPerformanceAnalyzer(BaseResultsAnalyzer):
     Get latency during operations performance analyzer
     """
 
-    def __init__(self, es_index, es_doc_type, email_recipients=(), logger=None, events=None):   # pylint: disable=too-many-arguments
+    def __init__(self, es_index, es_doc_type, email_recipients=(), logger=None, events=None,
+                 nemesis_class_name=None):   # pylint: disable=too-many-arguments
         super().__init__(es_index=es_index, es_doc_type=es_doc_type, email_recipients=email_recipients,
                          email_template_fp="results_latency_during_ops_short.html", logger=logger, events=events)
         self.percentiles = ['percentile_90', 'percentile_99']
-        self.test_name_for_email_subject = 'latency during operations'
+        self.test_name_for_email_subject = f'latency during {nemesis_class_name}'
```

Comment:
This thing should come from SCT configuration.

One should be able to put whatever he wants in the configuration for specific use case

Reply:
> This thing should come from SCT configuration.
> 
> One should be able to put whatever he wants in the configuration for specific use case

it does come from SCT configuration, but by taking the used `nemesis_class_name`

Reply:
I still don't understand why not using `email_subject_postfix` as the other perf test....

Reply:
I rather see it more align to the rest of the tests and with better control of each case

Reply:
here are 2 examples of email subject:
```
[aws][i3en.2xlarge] Performance Regression Compare Results (latency during upgrades 650gb) - test_latency_write_with_upgrade - 5.4.0~dev-0.20231006.498e3ec435be - 2023-10-11 19:09:57
```

and:
```
[aws][i3en.2xlarge] Performance Regression Compare Results (latency during operations 650gb) - test_latency_mixed_with_nemesis - 5.4.0~dev.20231003.d543b96d1837 - 2023-10-04 17:13:04
```

considering the extent of the subject already, to append something as a suffix (or prefix) will make it even longer, and some times, part of the important information is not shown in the list of emails